src/main/common/clients/launchDarklyClient.test.ts

```diff
@@ -0,0 +1,58 @@
+import config from 'config';
+import { when } from 'jest-when';
+import launchdarkly, { LDFlagValue } from 'launchdarkly-node-server-sdk';
+
+describe('LaunchDarkly', function () {
+  config.get = jest.fn();
+  const testFlag = 'test-flag';
+  jest.mock('config');
+  jest.mock('launchdarkly-node-server-sdk');
+
+  config.get = jest.fn();
+  //  launchDarklyClient = jest.fn();
+
+  let mockLdClient: {
+    waitForInitialization: () => Promise<unknown>;
+    variation: (flag: string, flagValue: LDFlagValue) => Promise<unknown>;
+  };
+
+  beforeEach(() => {
+    when(config.get).calledWith('featureToggles.launchDarklyKey').mockReturnValue('TEST_KEY');
+
+    when(launchdarkly.init).mockLdClient;
+    mockLdClient = {
+      // eslint-disable-next-line @typescript-eslint/no-explicit-any
+      waitForInitialization: async (): Promise<any> => {
+        undefined;
+      },
+      // eslint-disable-next-line @typescript-eslint/no-explicit-any
+      variation: async (): Promise<any> => Promise.resolve({ testFlag: true }),
+    };
+  });
+
+  // test('Should initiate ldUser and client', function () {
```

Comment:
Can we please remove these commented lines

Reply:
yes. they are removed in latest build now.